tools/StaticAnalysis/Exceptions/SignatureIssues.csv

```diff
@@ -1084,3 +1084,17 @@
 "Microsoft.Azure.Commands.Kubernetes.dll","Microsoft.Azure.Commands.Kubernetes.StartAzureRmKubernetesDashboard","Start-AzureRmKubernetesDashboard","1","8100","Start-AzureRmKubernetesDashboard Does not support ShouldProcess but the cmdlet verb Start indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.Commands.Kubernetes.dll","Microsoft.Azure.Commands.Kubernetes.StopAzureRmKubernetesDashboard","Stop-AzureRmKubernetesDashboard","1","8100","Stop-AzureRmKubernetesDashboard Does not support ShouldProcess but the cmdlet verb Stop indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.Commands.OperationalInsights.dll","Microsoft.Azure.Commands.OperationalInsights.Query.InvokeOperationalInsightsQuery","Invoke-AzureRmOperationalInsightsQuery","1","8100","Invoke-AzureRmOperationalInsightsQuery Does not support ShouldProcess but the cmdlet verb Invoke indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"Microsoft.Azure.Commands.IotHub.dll","Microsoft.Azure.Commands.Management.IotHub.NewAzureRmIotHubExportDevices","New-AzureRmIotHubExportDevices","1","8410","Parameter ExcludeKeys of cmdlet New-AzureRmIotHubExportDevices does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Microsoft.Azure.Commands.Sql.dll","Microsoft.Azure.Commands.Sql.VirtualNetworkRule.Cmdlet.NewAzureSqlServerVirtualNetworkRule","New-AzureRmSqlServerVirtualNetworkRule","2","8200","New-AzureRmSqlServerVirtualNetworkRule changes the confirm impact.  Please ensure that the change in ConfirmImpact is justified","Verify that ConfirmImpact is changed appropriately by the cmdlet. It is very rare for a cmdlet to change the ConfirmImpact."
```

Comment:
What are these SQL related exceptions? Was the build failing without them?

Reply:
Yes, the build was failing without adding these exception

Reply:
The SQL lines are marked as severity "2", which are recommendations. So, they will not stop the build from passing. Please remove those, and keep the IotHub exception.